cmd/policy/main.go

```diff
@@ -0,0 +1,397 @@
+		config.PolicyEngineName,
+		config.PolicyEngineParams,
+	)
+
+	if err != nil {
+		return fmt.Errorf("problem loading policy engine: %v", err)
+	}
+	defer rpcClient.Close()
+
+	policy, err := pm.GetPolicy(evidenceContext.TenantID, evidenceContext.Format)
+	if err != nil {
+		return fmt.Errorf(
+			"problem obtaining policy for tenant %d, format %q: %v",
+			evidenceContext.TenantID,
+			evidenceContext.Format.String(),
+			err,
+		)
+	}
+
+	err = pe.LoadPolicy(policy.Rules)
+	if err != nil {
+		return fmt.Errorf("problem loading policy into engine: %v", err)
+	}
+
+	var result common.AttestationResult
+
+	err = pe.GetAttetationResult(evidenceContext.Evidence, endorsements, false, &result)
+	if err != nil {
+		return fmt.Errorf("problem getting attestation result: %v", err)
+	}
+
+	resultBytes, err := json.MarshalIndent(result, "", "    ")
+	if err != nil {
+		return fmt.Errorf("problem encoding attestation result: %v", err)
+	}
+
+	fmt.Println(string(resultBytes))
+
+	return nil
+}
+
+var usageString = `
+Usage: %s [-c path/to/config/dir/] [-d] COMMAND [<command arg>...]
+
+Where COMMAND is one of the following:
+
+list
+    List all policies currently stored in the store. By default, tenant ID and
+    token format for all stored policies are listed. Optionally, -t flag may be
+    used to specify the tenant ID to which the output should be limited.
+
+get
+    Retrieves policies identified by the specified token formats (and,
+    optionally, tenant via -t) and outputs them as a zip archive.
+
+        [-t TENANT] [-o OUTFILE] FORMAT [FORMAT ...]
+
+    If tenant ID is not specified, it defaults to 1. If the name of the output
+    file is not specified, it defaults to "policy.zip" in the current
+    directory. See policy zip format description in the README for the
+    structure of the zip archive.
+
+add
+    Add policies from a zip file to the store. The following additional
+    arguments are supported:
+
+        [-t TENANT] [-f] POLICY_ZIP
+
+    The tenant ID is specified with -t (and defaults to 1, if not specified),
+    and POLICY_ZIP is the path to the zip file containing the policies (see
+    policy zip format in the README).
+
+    If a policy for the token format of one of the policies in the zip is
+    already present in the store, an error will be returned, unless -f is
+    specified, in which case, the existing policy will be overwritten.
+
+delete
+    Delete a policy identified by the specified token format (and, optionally,
+    tenant via -t).
+
+verify
+    Run a policy  against the specified endorsements and evidence context. This
+    can be used to test a policy to make sure it behaves as expected against a
+    known input. The following additional arguments are supported:
+
+	[-e ENDORSEMENTS] EVIDENCE_CONTEXT
+
+    where ENDORSEMENTS is the path to JSON-encoded endorsement values, and
+    EVIDENCE_CONTEXT is the path a file containing JSON-serialized
+    EvidenceContext.
+
+Top-level Options:
```

Comment:
looks like this should be the place where `-c` and `-d` are documented?

Reply:
These are generated by `flag.PrintDefaults()` in the `main()`.

Reply:
ah!  my bad, sorry for the silly comment.